app/views/pages/terms_of_use.html.erb

```diff
@@ -0,0 +1,57 @@
+
+    <h3>Warranty Disclaimers; Limitation of Liability</h3>
+    <p>The services are provided “as is” and without warranty of any kind. To the maximum extent permitted by law, the company disclaims all representations and warranties, express or implied, relating to the services or any content on the services, whether provided or owned by the company or by any third party, including without limitation warranties of merchantability, fitness for a particular purpose, title, non-infringement, freedom from computer virus, and any implied warranties arising from course of dealing, course of performance, or usage in trade, all of which are expressly disclaimed. In addition, you assume total responsibility and risk for your use of the services and the company does not make any representation or warranty that any of the services or any content available through any of the services is accurate, complete, available, current, free of viruses or other harmful components or defects, or that the services will meet your requirements. No advice or information, whether oral or written, obtained by you from company shall create any warranty not expressly made herein.</p>
+    <p>In no event whatsoever shall the company, its affiliates, or suppliers, or their respective officers, employees, shareholders, agents, or representatives, be liable for any indirect, consequential, incidental, special, punitive or exemplary damages, or for any loss of profits or revenue, including but not limited to loss of sales, profit, revenue, goodwill, or downtime, (arising under tort, contract, or other law) regardless of such party’s negligence or whether such party knew or should have known of the possibility of such damages. You understand and agree that the download of any materials in connection with the services is done at your discretion and risk and that you will be solely responsible for any loss or damage to your computer system or loss of data that may result from the download or upload of any material. Company neither assumes, nor does it authorize any other person to assume on its behalf, any other liability in connection with the provision of the services. If, notwithstanding the other provisions of these terms, company is found to be liable to you for any damage or loss which arises out of or is in any way connected with your use of any services, company’s liability shall in no event exceed the greater of (1) the total of any fees paid by you to company in the six (6) months prior to the date the claim is asserted for any of the services or feature relevant to the claim, or (2) US $500.00. These disclaimers and limitations of liability are made to the fullest extent permitted by law.</p>
```

Comment:
SHould we quote an amount in £ rather than $ ? And how do we work out what that amount is?

Reply:
I don't know, this is deffo the sketchiest part of the boilerplate (I flagged it in notion too) but I'm not keen on changing it if I don't know what I'm doing.

Deffo part of a review process for later to look at this small print.